### PR TITLE
Added Game Network rulesets

### PR DESCRIPTION
**Pull request by RemakingEden**
Closes https://github.com/EFForg/https-everywhere/issues/17082

### CODE REVIEW

File: src/chrome/content/rules/Nintendolife.com.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+	Nonfunctional hosts in *.nintendolife.com:
+
+# All sites stay as http with this ruleset active.
```
Comment (pipboy96) [inline src/chrome/content/rules/Nintendolife.com.xml]: Can you explain what do you mean by this?

File: src/chrome/content/rules/Nintendolife.com.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+-->
+<ruleset name="Nintendolife.com">
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Nintendolife.com.xml]: There's no need for a line feed here.

File: src/chrome/content/rules/Pushsquare.com.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!--
+	Nonfunctional hosts in *.pushsquare.com:
+
+# All sites remain on http with this ruleset
+-->
+<ruleset name="Pushsquare.com">
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Pushsquare.com.xml]: There's no need for a line feed here.

File: src/chrome/content/rules/Nintendolife.com.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <target host="ds.nintendolife.com" />
+    <target host="images.nintendolife.com" />
+    <target host="m.nintendolife.com" />
+    <target host="retro.nintendolife.com" />
+    <target host="static.nintendolife.com" />
+    <target host="vc.nintendolife.com" />
+    <target host="wii.nintendolife.com" />
+    <target host="wiiu.nintendolife.com" />
+    <target host="wiiware.nintendolife.com" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Nintendolife.com.xml]: Add `<securecookie host=".+" name=".+" />` here.

File: src/chrome/content/rules/Pushsquare.com.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <target host="pushsquare.com" />
+    <target host="www.pushsquare.com" />
+    <target host="images.pushsquare.com" />
+    <target host="static.pushsquare.com" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Pushsquare.com.xml]: Add `<securecookie host=".+" name=".+" />` here.

File: src/chrome/content/rules/Nintendolife.com.xml (hunk 4 above)
Comment (RemakingEden) [reply src/chrome/content/rules/Nintendolife.com.xml]: Can you explain what this does please?

File: src/chrome/content/rules/Nintendolife.com.xml (hunk 1 above)
Comment (RemakingEden) [reply src/chrome/content/rules/Nintendolife.com.xml]: All the sites stay as http:// none of them redirect to https://

Comment (pipboy96) [reply src/chrome/content/rules/Nintendolife.com.xml]: It's fine to remove the entire comment then.

File: src/chrome/content/rules/Nintendolife.com.xml (hunk 4 above)
Comment (pipboy96) [reply src/chrome/content/rules/Nintendolife.com.xml]: https://github.com/EFForg/https-everywhere/blob/master/docs/en_US/rulesets.md#secure-cookies

**Comment by pipboy96** [comment]
Please make sure the tests pass.

File: src/chrome/content/rules/Pushsquare.com.xml (hunk 5 above)
Comment (RemakingEden) [reply src/chrome/content/rules/Pushsquare.com.xml]: Thanks for the help mate

File: src/chrome/content/rules/Pushsquare.com.xml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="Pushsquare.com">
+
+    <target host="pushsquare.com" />
+    <target host="www.pushsquare.com" />
+    <target host="images.pushsquare.com" />
+    <target host="static.pushsquare.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Pushsquare.com.xml]:
```
fs.pushsquare.com
sp.pushsquare.com
```

File: src/chrome/content/rules/Nintendolife.com.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    <target host="3dsvc.nintendolife.com" />
+    <target host="3dsware.nintendolife.com" />
+    <target host="dsiware.nintendolife.com" />
+    <target host="ds.nintendolife.com" />
+    <target host="images.nintendolife.com" />
+    <target host="retro.nintendolife.com" />
+    <target host="static.nintendolife.com" />
+    <target host="vc.nintendolife.com" />
+    <target host="wii.nintendolife.com" />
+    <target host="wiiware.nintendolife.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Nintendolife.com.xml]:
```
fs.nintendolife.com
sp.nintendolife.com
```

**Comment by RemakingEden** [comment]
> Maybe you're also interested to add a ruleset for `gamer-network.net` itself?

I will look into this now :)

File: src/chrome/content/rules/Nintendolife.com.xml (hunk 7 above)
Comment (RemakingEden) [reply src/chrome/content/rules/Nintendolife.com.xml]: Those two url's do not resolve for me? That is why I removed them. Do they work for you?

Comment (J0WI) [reply src/chrome/content/rules/Nintendolife.com.xml]: They WFM and some resources are loaded from there.

**Pull request merged**